src/components/HTMLEngineProvider/HTMLRenderers/MentionUserRenderer.tsx

```diff
@@ -76,14 +82,14 @@ function MentionUserRenderer(props) {
                         }}
                     >
                         <Text
-                            style={[styles.link, _.omit(props.style, 'color'), StyleUtils.getMentionStyle(isOurMention), {color: StyleUtils.getMentionTextColor(isOurMention)}]}
+                            // eslint-disable-next-line react/jsx-props-no-spreading
+                            {...defaultRendererProps}
+                            style={[styles.link, styleWithoutColor, StyleUtils.getMentionStyle(isOurMention) as StyleProp<TextStyle>, {color: StyleUtils.getMentionTextColor(isOurMention)}]}
```

Comment:
Update `getMentionStyle` return type to TextStyle
```suggestion
                            style={[styles.link, styleWithoutColor, StyleUtils.getMentionStyle(isOurMention), {color: StyleUtils.getMentionTextColor(isOurMention)}]}
```

Reply:
Still relevant